Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8.3
+ENV PROJECT_DIR=/brew-control-dash
+ENV PATH="${PATH}:/root/.poetry/bin"
+WORKDIR $PROJECT_DIR
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
```

Comment:
i had to add e.g `--retry 5` to this the other day to avoid "connection dropped" errors bonking the container build. no biggie if this is working consistently